docs/installation/install-node.rst

```diff
@@ -44,6 +44,54 @@ If you do not have any nodes, the following guides will help you set some up, wi
 
 .. XXX In the integration specific documentation, links to the guides appear here
 
+.. begin-body-installing-node-rhel
+
+Installing on RHEL 7
+====================
+
+.. note:: You should ensure your nodes are Flocker-ready, either by checking the prerequisites above, or by following our guides on using Amazon Web Services or Rackspace.
+
+#. **Log into the first node as root:**
+
+   .. prompt:: bash alice@mercury:~$
+
+      ssh root@<your-first-node>
+
+#. **Install the** ``clusterhq-flocker-node`` **package:**
+
+   To install ``clusterhq-flocker-node`` on RHEL 7 you must install the RPM package provided by the ClusterHQ repository.
+   The commands below will install the two repositories and the ``clusterhq-flocker-node`` package.
+   
+   Run the following commands as root on the target node:
+
+   .. prompt:: bash root@rhel:~$
```

Comment:
If this is run as root, should the prompt directive be `.. prompt:: bash [root@rhel]#`?